sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4404,10 +4404,12 @@ object SQLConf {
         s"the $SESSION_CATALOG_NAME and must be consistent with it; for example, if a table can " +
         s"be loaded by the $SESSION_CATALOG_NAME, this catalog must also return the table " +
         s"metadata. To delegate operations to the $SESSION_CATALOG_NAME, implementations can " +
-        "extend 'CatalogExtension'.")
+        "extend 'CatalogExtension'. The value should be either 'builtin' which represents the " +
+        "spark's builit-in V2SessionCatalog, or a fully qualified class name of the catalog " +
+        "implementation.")
       .version("3.0.0")
       .stringConf
-      .createOptional
+      .createWithDefault("builtin")
```

Comment:
shall we normalize it to lower case? e.g. `.transform(_.toLowerCase(Locale.ROOT))`

Reply:
thanks for suggestion, updated

Reply:
sorry I'm wrong... It's class name so we can't lower case it.

Maybe the string comparisons should be case insensitive.

Reply:
oh, right, let me change

Reply:
@cloud-fan how about now?